uphold-integration/src/main/java/org/dash/wallet/integration/uphold/data/UpholdClient.java

```diff
@@ -162,8 +206,8 @@ public void onResponse(Call<List<UpholdCard>> call, Response<List<UpholdCard>> r
                         }
                     }
                 } else {
-                    log.error("Error to obtain cards", response.message());
-                    callback.onError(new Exception(response.message()), false);
+                    log.error("Error obtaining cards " + response.message());
+                    callback.onError(new UpholdException("Error obtaining cards", response.message(), response.code()), false);
```

Comment:
The UpholdException class was created to send the response codes to the callback methods.

Reply:
unless that last parameter of the log methods is an Exception.